action.yml

```
@@ -100,7 +100,8 @@ runs:
           echo "Installing from git branch: $GITHUB_REF"
           packages="git+https://github.com/ultralytics/actions@${GITHUB_REF#refs/heads/}"
         else
-          packages="ultralytics-actions"
+          # packages="ultralytics-actions"
+          "git+https://github.com/ultralytics/actions@main"
```

❗ **CRITICAL**: `packages` no longer receives a value in the default branch path. Later when we call `pip install "$packages"`, the variable will be empty and the install step will fail. Please restore the assignment.

**Suggested change:**
```suggestion
          packages="git+https://github.com/ultralytics/actions@main"
```